nova/scheduler/turbonomic_scheduler.py

```diff
@@ -19,33 +19,33 @@
 
 For integrating this scheduler to get Placement recommendations,
 the following entries must be added in the /etc/nova/nova.conf file
-under the [scheduler] section
+under the [DEFAULT] section
 ------------------------------------------------------------
-driver = nova.scheduler.turbonomic_scheduler.TurbonomicScheduler
+scheduler_driver = nova.scheduler.turbonomic_scheduler.TurbonomicScheduler
```

Comment:
This wasnt changed in the Mitaka branch - Why is this showing up as a change?
This exists only on the Pike branch - Did you create the patch on the right branch?